tst/unit/test_state_descriptor.cpp

```diff
@@ -383,16 +383,16 @@ TEST_CASE("Test dependency resolution in StateDescriptor", "[StateDescriptor]")
       }
     }
 
-    WHEN("We register a dense variable custom prolongation/restriction") {
+    WHEN("We register a dense variable with default prolongation/restriction") {
```

Comment:
These are still custom right? Not default?

Reply:
I don't think so... only the `SparsePool` has enrolled custom P/R ops via `RegisterRefinementOps`.  The dense field just uses the default ones.

Reply:
Oh yes you're right. :+1: